.github/workflows/workflow.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Upload artifacts
         uses: actions/upload-artifact@master
-        if: matrix.os == 'ubuntu-latest'
+        if: matrix.os == 'ubuntu-latest' && matrix.node == 16
         with:
           name: vscode-lit-plugin
-          path: vscode-lit-plugin
+          path: vscode-lit-plugin # is this right? the actual path is packages/vscode-lit-plugin
```

Comment:
There's some script that builds the plugin to a top-level folder. Documenting and simplifying the build system stuff should be one of the priorities.